### PR TITLE
Loading indices to ES

### PR DESCRIPTION
**Pull request by goru97**
*Why:*
To fix the error given by BF on Querying for the List of metrices.

### CODE REVIEW

File: contrib/blueflood-docker/configs/ES-Setup/index_settings.json [hunk 1]
```diff
@@ -0,0 +1,27 @@
+{
```
Comment (ChandraAddala) [inline contrib/blueflood-docker/configs/ES-Setup/index_settings.json]: Is there anyway we can avoid having multiple copies of these files?

Comment (ChandraAddala) [inline contrib/blueflood-docker/configs/ES-Setup/index_settings.json]: The original one is located at ./blueflood-elasticsearch/src/main/resources/index_settings.json

Comment (goru97) [inline contrib/blueflood-docker/configs/ES-Setup/index_settings.json]: @ChandraAddala I know, but I wanted the blueflood-docker repo to be independent of other packages.

Comment (ChandraAddala) [inline contrib/blueflood-docker/configs/ES-Setup/index_settings.json]: If the goal to is to keep this independent, I guess it makes sense, but I still feel if these files change, someone has to remember to change them here.

**Comment by goru97** [comment]
@ChandraAddala You are right, but these files rarely change as they define schemas in ES and the other files are configs (blueflood.conf, properties), which people can adjust to taste.

File: contrib/blueflood-docker/configs/ES-Setup/index_settings.json (hunk 1 above)
Comment (usnavi) [inline contrib/blueflood-docker/configs/ES-Setup/index_settings.json]: Does this need to be independent?  Is there a use case for having our docker example load up a different schema than our actually install?

File: contrib/blueflood-docker/configs/ES-Setup/init-es.sh [hunk 2]
```diff
@@ -0,0 +1,79 @@
+#!/bin/bash
```
Comment (usnavi) [inline contrib/blueflood-docker/configs/ES-Setup/init-es.sh]: Is this copied from blueflood-elasticsearch/src/main/resource/init-es.sh?  Same copy questions as before.

Comment (goru97) [inline contrib/blueflood-docker/configs/ES-Setup/init-es.sh]: Yes! These are copied from blueflood-elasticsearch/src/main/resource/init-es.sh. Schema rarely changes; it's the blueflood.conf which people usually modify.

**Comment by usnavi** [comment]
@goru97 - let's have a quick chat today and see if we can reference some of hte existing scripts.

**Comment by goru97** [comment]
@ChandraAddala @usnavi Can I merge this for now. I will write a new dockerfile, which will be more according to the standards, very configurable and will also use the resources which are already there in ./blueflood-elasticsearch/src/main/resources. But people are using this repo and are saying it's broken!

**Comment by usnavi** [comment]
Sure, you can merge it now....as long as you promise.... :-P

**Comment by goru97** [comment]
I keep my promises :smiley:

**Pull request merged**